app/controllers/people_controller.rb

```diff
@@ -0,0 +1,18 @@
+require 'pry'
```

Comment:
You should avoid committing files with `require 'pry'` in them as the code will fail when deployed to Heroku (which will not have pry installed). The same goes for any `binding.pry` lines as well.

The `require` line is a lot easier to forget to remove, but Rails will actually sense that you have the pry gem installed and do that part for you, so you won't need to add those lines at all :)